resources/assets/js/components/StatsImpact.vue

```diff
@@ -7,29 +7,33 @@
       </span>
     </h2>
     <div class="impact-container">
-      <EventStatsValue :count="stats.ewaste + stats.unpowered_waste" icon="trash" size="md" title="partials.waste_prevented" unit="kg" class="impact-waste" />
+      <StatsValue :count="stats.ewaste + stats.unpowered_waste" icon="trash" size="md" title="partials.waste_prevented" unit="kg" class="impact-waste" />
       <div v-if="notincluded" class="d-flex justify-content-end">
         <div class="impact-notincluded">
           <div class="impact-notincluded-content p-1">
             {{ notincluded }}
           </div>
         </div>
       </div>
-      <EventStatsValue :count="stats.co2" icon="cloud_empty" size="lg" title="partials.co2" subtitle="partials.powered_only" :description="equivalent_consumer(stats.co2)" unit="kg" class="impact-co2" />
+      <StatsValue :count="stats.co2" icon="cloud_empty" size="lg" title="partials.co2" subtitle="partials.powered_only" :description="equivalent_consumer(stats.co2)" unit="kg" class="impact-co2" />
     </div>
   </div>
 </template>
 <script>
-import EventStatsValue from './EventStatsValue'
+import StatsValue from './StatsValue'
 import co2equivalent from '../mixins/co2equivalent'
 
 export default {
-  components: {EventStatsValue},
+  components: {StatsValue},
   mixins: [ co2equivalent ],
   props: {
     stats: {
       required: true,
       type: Object
+    },
+    statsEntity: {
+      required: true,
+      type: String
     }
```

Comment:
FYI, there isn't (so far as I know) a built-in "Enum" type.  If you're being super-careful, you can add a custom validator to enforce the usage.  I don't think you need to do that here but it's worth skimming the way it's done.

https://vuejs.org/v2/guide/components-props.html#Prop-Validation